provisioning/stream/docker/k6/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM grafana/xk6:latest AS build
 
 RUN xk6 build \
     --with github.com/LeonAdato/xk6-output-statsd@latest \
-    --with github.com/domsolutions/xk6-fasthttp@latest
+    --with github.com/domsolutions/xk6-fasthttp@0.1.1
```

Comment:
Latest version of `github.com/domsolutions/xk6-fasthttp` requires Go 1.23 but `grafana/xk6` Dockerfile is built from Go `1.22.7`.